### PR TITLE
Add Flutter wrapper to our project.

### PR DESCRIPTION
**Pull request by jeffs-sf**
## Why?

Candidates can be on various versions of flutter based on other projects they are working on, things they are trying out, etc. This can lead to some headache when we are trying to run the submission on different machines and environments.

## What?

Add a wrapper to this project utilizing https://github.com/passsy/flutter_wrapper.
Add documentation informing the candidate how to use this wrapper within the project.

## Types of Changes

New feature (non-breaking change which adds functionality)

## Documentation

More information can be found on https://github.com/passsy/flutter_wrapper.

## Validation

Ensure the project is set up to use the wrapped version of Flutter which allows all devs working on the project to be forced to the same version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,51 @@
...
+
+- `./flutterw channel {x}` to switch the Flutter channel.
+- `./flutterw upgrade` to upgrade the version of Flutter.
+- `./flutterw run -d {device_id}` to run the app.
+
+### IDE Setup
+
+<details>
+<summary>Use with VSCode</summary>
+<p>
```
Comment (jeffs-sf) [inline README.md:23]: **info:** these are collapsibles since some might already be familiar with the approach. I didn't want to push down the rest of the content too far.

File: README.md [hunk 2]
```diff
@@ -51,7 +96,7 @@ Feel free to add more tests as you see fit but the above is the minimum requirem
 ## Design
 
 - See this [Figma File](https://www.figma.com/file/UOQDbU02GG2yaJMfrO9q9d/Flutter-Test?node-id=0%3A1) for design specifics like
-fonts, themes, colors, etc.
+  fonts, themes, colors, etc.
```
Comment (jeffs-sf) [inline README.md:99]: **info:** auto formatted. no diff.

File: README.md [hunk 3]
```diff
@@ -111,7 +156,7 @@ While the purpose of this challenge is not to guage whether you can achieve 100%
 If you are feeling up to it and want to add some more functionality to the application try some of these bonus items
 
 - Implement persistance storage for favorited restaurants
-- Implement the *Bonus Detail Screen*
+- Implement the _Bonus Detail Screen_
```
Comment (jeffs-sf) [inline README.md:159]: **info:** auto formatted. no diff.

**Review by briblanch — APPROVED** [review]

**Review by brianb-sf — APPROVED** [review]
+1 😄
